portal/src/components/ExampleVideo/ExampleVideo.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC, useRef, useState } from "react";
+import classNames from "classnames";
+import "./ExampleVideo.scss";
+
+interface Props {
+    videoUrl: string;
+}
+
+export const ExampleVideo: FC<Props> = ({ videoUrl }) => {
+    const urlWithParams = videoUrl + "?background=1&autoplay=0&loop=1";
+    const [playing, togglePlayState] = useState(false);
+    const ref = useRef<HTMLIFrameElement>(null);
+
+    const togglePlay = () => {
+        if (ref.current) {
+            const method = playing ? "pause" : "play";
+            const playerData = { method: method };
+            ref.current.contentWindow?.postMessage(JSON.stringify(playerData), "*");
+            togglePlayState(!playing);
+        }
+    };
+
+    const buttonClass = classNames("example-video__button", {
+        "example-video__button--play": !playing,
+        "example-video__button--pause": playing,
+    });
+
+    return (
+        <div className="example-video">
+            <button className={buttonClass} onClick={togglePlay} />
```

Comment:
Denne trenger en tittel eller annen form for henvisning til hva den gjør, som også må ta hensyn til state.

Reply:
Fixed